### DISCUSSION TITLE
Suggestion Collection - Mod Workshop

### DISCUSSION THREAD

**Discussion by SenorRagequit** [exorstudios/riftbreaker-tools#236] [Wishlist] [answered]
This discussion shall be a guiding light in creating the Riftbreaker mod workshop. 
Nothing in here is mandatory for the mod workshop to have, but many ideas will forge a great product. 

What's something the Riftbreaker internal mod workshop should have in your opinion?

**Comment by SenorRagequit** [comment]
**My priorities would be:**

### As a mod dev
- Set mod name
- Set mod version
- Set mod description
- Set pre-created mod tags (so no abuse), like Monsters, Buildings, Weather, Map etc.
- Set mod author + co-authors
- Set requirements (DLC stuff)
- Being able to see the numbers of downloads
- Set a thumbnail + max 5 other pics
- Some kind of patching + changelog functionality
- Link to some Youthbe video explaining it / showing it off maybe ?

### As a mod user
- See mod name
- See mod description
- See mod author + co-authors
- See number of mod downloads
- Some kind of rating (still unsure regarding abuse)
- Being able to hide a mod
- Report a mod
- Main categories (Hot / New) + some general overview
- Filter mods by pre-created mod tags like Monsters, Buildings, Weather, Map etc.
- Being able to directly report mod bugs maybe?

**Comment by Nino-Bock** [comment]
My wish/suggestion would be to allow mod authors to add some documentation into the game.

Some more complex mods like Expanded Arsenal for example have much interaction going on.
Like for example which weapon type to use against which creature.

What I don't like is, if a game needs an external source of information like a wiki or Nexus in order to understand the mod completely.
Mod authors should be given some place inside the game to make all the needed information accessible to the player.

**Comment by SenorRagequit** [comment]
What about a way to link to mods? Some kind of internal Riftbreaker URL and when you click it you get to see the mod. A mod dev can share that link or users send it to another so they dont even have to search it

**Comment by Nino-Bock** [comment]
I do not like such a solution because I can never be sure hoe long such external sources are available.

I purchase my games from GOG and keep offline backups.
to me it is important to be able to play the games in a few years and keep my collection as future proof as possible.

So if I have the game and the mod stored offline, but essential information is missing, that is not optimal for me.

Also I am playing on a handheld and Internet is not always available.

Having all the needed information available in game is the best solution IMHO.

**Reply by SenorRagequit** [reply]
> I do not like such a solution because I can never be sure hoe long such external sources are available.
> 
> I purchase my games from GOG and keep offline backups. to me it is important to be able to play the games in a few years and keep my collection as future proof as possible.
> 
> So if I have the game and the mod stored offline, but essential information is missing, that is not optimal for me.
> 
> Also I am playing on a handheld and Internet is not always available.
> 
> Having all the needed information available in game is the best solution IMHO.

Mate, I am talking about the topic of this thread, not what you said. What you said is a valid statement.

**Reply by Nino-Bock** [reply]
> > I do not like such a solution because I can never be sure hoe long such external sources are available.
> > I purchase my games from GOG and keep offline backups. to me it is important to be able to play the games in a few years and keep my collection as future proof as possible.
> > So if I have the game and the mod stored offline, but essential information is missing, that is not optimal for me.
> > Also I am playing on a handheld and Internet is not always available.
> > Having all the needed information available in game is the best solution IMHO.
> 
> Mate, I am talking about the topic of this thread, not what you said. What you said is a valid statement.

I'm sorry I got you wrong :)

**Comment by SenorRagequit** [comment, marked as answer]
We got it in game now.
